analyses/cell-type-ewings/04-infercnv.Rmd

```diff
@@ -0,0 +1,433 @@
+    }) |>
+  # concatenate vertically into HeatmapList object
+  purrr::reduce(ComplexHeatmap::`%v%`) |>
+  ComplexHeatmap::draw(
+    heatmap_legend_side = "right",
+    # add a margin to the heatmap so labels don't get cut off
+    padding = unit(c(2, 20, 2, 2), "mm")
+  )
+```
+
+This looks as we expect with mostly muscle cells lining up with tumor cells. 
+
+## Check for known CNVs in Ewing's 
+
+There are a few known copy number variations in Ewing's sarcoma: 
+
+- Gain of Chr8
+- Gain of Chr12
+- Gain of Chr1p
+- Loss of Chr16q
+
+Although these are the most frequent, there are patients who do not have any of these alterations and patients that only have some of these alterations. 
+See [Tirode et al.,](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264969/) and [Crompton et al.](https://doi.org/10.1158/2159-8290.CD-13-1037). 
+
+Below we will plot whether or not the gain or loss is present in each cell according to `InferCNV`. 
+
+```{r}
+# umaps showing where cells are with gains/losses 
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = has_dupli_chr8)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr8 gain") +
+  theme_bw()
+
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = has_dupli_chr12)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr12 gain") +
+  theme_bw()
+
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = has_dupli_chr1)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr1 gain") +
+  theme_bw()
+
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = has_loss_chr16)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr16 loss") +
+  theme_bw()
+```
+
+From this we see that the group of tumor cells at the bottom have chr8 and chr1 gains with some having gains of chr12. 
+It doesn't look like any cells have loss of chr16. 
+The other interesting thing is that there is a group of cells that have gains of chr8, chr1, and chr12 that we did not classify as tumor cells. Should they be... 
+
+We can also look at the proportion of the chromosome with the gain. 
+
+```{r}
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = proportion_scaled_dupli_chr8)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr8 gain") +
+  theme_bw() +
+  scale_color_viridis_c()
+
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = proportion_scaled_dupli_chr12)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr12 gain") +
+  theme_bw() +
+  scale_color_viridis_c()
+
+ggplot(cnv_df, aes(x= UMAP1, y = UMAP2, color = proportion_scaled_dupli_chr1)) +
+  geom_point(alpha = 0.5, size = 0.5) +
+  labs(title = "Chr1 gain") +
+  theme_bw() +
+  scale_color_viridis_c()
+```
+
+Now when we look at the proportion, we see that only the group of tumor cells on the bottom seem to have a higher proportion of genes in chr8 that have a gain in expression. 
+Based on the papers, we should see a gain in the whole chr8 and chr12, not just one arm or specific genes, so we expect a high value for the proportion. 
```

Comment:
This makes sense to me, as it seems like the method has a high chance of calling small CNVs somewhere in the chromosome. This is why I would expect that the proportion is going to be a better measure overall. There may also be some value in looking at the _variance_ in the proportions across chromosomes. We might expect that tumor cells have greater variance based on the fact that they are likely to have some larger duplications/deletions than normal cells.